api/server.go

```diff
@@ -65,7 +69,7 @@ func StartServer() {
 			throttled.DefaultDeniedHandler.ServeHTTP(w, r)
 		})
 
-		handler = th.Throttle(Srv.Router)
+		handler = th.Throttle(&CorsWrapper{Srv.Router})
```

Comment:
I think it will only add the CORS stuff if you have `EnableRateLimiter` enabled ?